src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -1287,8 +1287,8 @@ public partial class BinaryWriter : System.IDisposable
         public virtual void Write(uint value) { }
         [System.CLSCompliantAttribute(false)]
         public virtual void Write(ulong value) { }
-        public virtual void Write(System.ReadOnlySpan<byte> span) { }
-        public virtual void Write(System.ReadOnlySpan<char> span) { }
+        public virtual void Write(System.ReadOnlySpan<byte> buffer) { }
+        public virtual void Write(System.ReadOnlySpan<char> chars) { }
```

Comment:
Should the argument name be buffer here instead of chars, similar to:
```C#
public virtual int Read(System.Span<char> buffer) { throw null; } 
```

Reply:
I'm matching the argument name used above in:
```C#
         public virtual void Write(char[] chars) { }
         public virtual void Write(char[] chars, int index, int count) { }
```

Reply:
I see. That makes sense. It looks like we don't have consistency in the naming between the BinaryReader and BinaryWriter.
```C#
public virtual int Read(char[] buffer, int index, int count) { throw null; }
```

Reply:
Yeah.  Unfortunately when something's already inconsistent, there's little you can do other than not make it worse :)